pkg/errors/errors_test.go

```diff
@@ -67,31 +59,66 @@ func TestContains(t *testing.T) {
 		desc      string
 		container error
 		contained error
+		contains  bool
 	}{
+		{
+			desc:      "nil contains nil",
+			container: nil,
+			contained: nil,
+			contains:  true,
```

Comment:
I would use `err`, `contained` and `res` or something easier to differentiate

Reply:
IMO, this is a nice triplet. You know instantly which error contains, which error is contained and what should be result.

Reply:
@manuio I also use this "terminology" in `TestWrap` and also a similar double term like wrapper and wrapped. IMO, it's very clear what this all means, while `err` and `res` are too generic.